components/submit-expense/useExpenseForm.ts

```diff
@@ -1424,7 +1425,19 @@ async function buildFormOptions(
 
     if (account?.supportedExpenseTypes) {
       options.supportedExpenseTypes = account.supportedExpenseTypes;
-      if (!supportsBaseExpenseTypes(options.supportedExpenseTypes) && !query.loading) {
+      if (
+        values.expenseTypeOption === ExpenseType.GRANT &&
+        account.type === CollectiveType.FUND &&
+        !options.supportedExpenseTypes.includes(ExpenseType.GRANT)
+      ) {
+        options.supportedExpenseTypes.push(ExpenseType.GRANT);
+      }
+
```

Comment:
I don't understand that one; why would we override the `supportedExpenseTypes` from API? If it is wrong, then the fix should be in the API.